sheetScript.gs

```diff
@@ -22,6 +22,25 @@ var reservedColumnDeleteDays = 3; // Column containing number of days before tod
 // If you use getRange remember those indices are one-based, so add + 1 in that call i.e.
 // sheet.getRange(iRow + 1, reservedColumnTimestamp + 1).setValue(isodate);
 
+// Extend Date with Iso String with timzone support (Youtube needs IsoDates)
+// https://stackoverflow.com/questions/17415579/how-to-iso-8601-format-a-date-with-timezone-offset-in-javascript
+Date.prototype.toIsoString = function() {
+    var tzo = -this.getTimezoneOffset(),
```

Comment:
Since making a copy of the sheet doesn't change script's timezone (or the sheet's timezone), users will need to manually change it. Add note to readme?

![image](https://user-images.githubusercontent.com/17478849/80022886-b38eac00-84aa-11ea-8720-d486ca03bcea.png)